### ISSUE TITLE
Dropdown component - clear filter button is not accessible via keyboard

### ISSUE THREAD

**Issue by immanenipavankumar** [IBM/carbon-components-angular#1102]
> Describe in detail the issue you're having.

Our AVT team flagged that the clear filter button cannot be accessed by keyboard, pressing Tab doesn't take the control over to the button.

> Is this a feature request (new component, new icon), a bug, or a general issue?

Bug

> Is this issue related to a specific component?

Dropdown 

> What did you expect to happen? What happened instead? What would you like to see changed?

> What browser are you working in?

Chrome

> What version of the Carbon Design System are you using?

V3.40.4

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

IBM ADDI
